core/utils.js

```diff
@@ -834,3 +834,28 @@ Blockly.utils.insertAfter_ = function(newNode, refNode) {
     parentNode.appendChild(newNode);
   }
 };
+
+/**
+ * Calls a function after the page has loaded, possibly immediately.
+ *
+ * This function does not run in a Node.js environment due to lack of document
```

Comment:
This comment ("This function does not run...) belongs at line 92 of extensions.js, with a note that in that case validation will not occur until the first block of that type is constructed.

The fact that it'll break in node.js is covered by the @throws annotation.